reports/tests/test_views.py

```diff
@@ -1,7 +1,266 @@
-# Could not get this work yet:
-# - "axes.exceptions.AxesBackendRequestParameterRequired:
-# AxesBackend requires a request as an argument to authenticate"
```

Comment:
The `AxesBackend` used in our API in order to prevent brute force attacks to `django-admin`.
There seems to be a [compatible issue](https://github.com/jazzband/django-axes/issues/340) with pytest `admin_client` fixture and the `AxesBackend` authentication backend.
I think the quickest solution here is to remove the `"axes.backends.AxesBackend"` from `AUTHENTICATION_BACKENDS` test setting (using `override_settings`) when running the test.

Reply:
Good to know! But I think I will still jump over this test at this time.